### ISSUE TITLE
Lix download on post-install may fail due to lix not being in PATH

### ISSUE THREAD

**Issue by gene-pavlovsky** [elsassph/haxe-modular#111]
This line broke our CI: https://github.com/elsassph/haxe-modular/blob/master/package.json#L10
I don't think it's safe to assume that lix is in PATH in an npm post-install script.
Our build installs both lix and modular via npm, and only then adds node_modules/.bin to PATH.
There is no globally installed lix on build agents, and in my opinion modular shouldn't make assumptions like that in it's postinstall script.

**Comment by kLabz** [comment]
No it's fine. `lix` doesn't need to be in PATH when called from within a npm script, it will resolve from node_modules/.bin

Just double-checked in a temp directory (while not having lix in PATH ofc):

```
$ cat package.json 
{
  "name": "test",
  "version": "0.1.0",
  "description": "",
  "scripts": {
    "postinstall": "lix download"
  },
  "dependencies": {},
  "devDependencies": {
    "lix": "^15.10.1"
  }
}

$ npm i

> lix@15.11.4 postinstall /tmp/tmp.lrwqz0bspI/node_modules/lix
> node bin/postinstall.js


> test@0.1.0 postinstall /tmp/tmp.lrwqz0bspI
> lix download

Downloading Haxe: official release 4.2.3
-> Done!
```

**Comment by gene-pavlovsky** [comment]
Well there seems to be some condition when it doesn't work like that. I'm getting this error:
`npm ERR! haxe-modular@0.13.0 postinstall: `lix download``

**Comment by kLabz** [comment]
Hmm yeah, doesn't work (and shouldn't be called) when installing haxe-modular as a dependency.

**Comment by elsassph** [comment]
Yes it's my mistake. It should be a `prepare` task which won't run when installing Modular as a dependency..

**Comment by elsassph** [comment]
I published `0.13.1` without the `postinstall` task.

**Comment by gene-pavlovsky** [comment]
Thanks Philippe, I will check it out next week
